app/Renderers/DatagridRenderer.php

```diff
@@ -25,6 +25,7 @@ class DatagridRenderer
 
     protected string $hidden = ' hidden lg:table-cell';
     protected array $columns = [];
+    protected null|int $bookmark = null;
```

Comment:
Could save the whole bookmark object in case in the future we can to do something with it (and no need for a null version, that can be checkec with isset())